contracts/examples/AyiiProduct.sol

```diff
@@ -463,18 +476,75 @@ contract AyiiProduct is
     function getRiskId(uint256 idx) external view returns(bytes32 riskId) { return _riskIds[idx]; }
     function getRisk(bytes32 riskId) external view returns(Risk memory risk) { return _risks[riskId]; }
 
+    function applications() external view returns(uint256 applicationCount) {
+        return _applications.length;
+    }
+
+    function getApplicationId(uint256 applicationIdx) external view returns(bytes32 processId) {
+        return _applications[applicationIdx];
+    }
+
     function policies(bytes32 riskId) external view returns(uint256 policyCount) {
         return EnumerableSet.length(_policies[riskId]);
     }
 
-    function getPolicyId(bytes32 riskId, uint256 policyIdx) external view returns(bytes32 policyId) {
+    function getPolicyId(bytes32 riskId, uint256 policyIdx) external view returns(bytes32 processId) {
         return EnumerableSet.at(_policies[riskId], policyIdx);
     }
 
     function getApplicationDataStructure() external override pure returns(string memory dataStructure) {
         return "(bytes32 riskId)";
     }
 
+
+    function _validateRiskParameters(
```

Comment:
i think it would be valuable to document the full names of these parameters and their function somewhere in the class.

Reply:
added description of parameter top level at the Risk struct definition